### PR TITLE
added github action for publishing package to pypi

### PR DESCRIPTION
**Pull request by ruscooler**

### CODE REVIEW

File: .github/workflows/create_pypi_package.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        with:
+          virtualenvs-create: false
+
+      - name: Patch package version
+        run: poetry version patch
+
+      - name: Run Dockerfile to build and publish package into PyPI
+        run: |
+          docker build . -t ignored \
+            --build-arg PYPI_USERNAME=${{ secrets.PYPI_USERNAME }} \
```
Comment (DementevNikita) [inline .github/workflows/create_pypi_package.yaml:25]: It'd be great to add those secrets as organization's and remove them as repository's for each project that uses them.
* oddrn-generator
* odd-models-package

File: .github/workflows/create_pypi_package.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+name: Build and publish PyPI artifact
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build_and_publish_package:
+    name: Build and publish the PyPi package
```
Comment (DementevNikita) [inline .github/workflows/create_pypi_package.yaml:8]: Please add concurrency

**Review by DementevNikita — CHANGES_REQUESTED** [review]

File: .github/workflows/create_pypi_package.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+name: Build and publish PyPI artifact
+
+on:
+  workflow_dispatch:
```
[the remark this answers is not in the document]
Comment (ruscooler) [reply .github/workflows/create_pypi_package.yaml]: done

File: .github/workflows/create_pypi_package.yaml (hunk 1 above)
Comment (ruscooler) [reply .github/workflows/create_pypi_package.yaml:28]: done

File: .github/workflows/create_pypi_package.yaml (hunk 2 above)
Comment (ruscooler) [reply .github/workflows/create_pypi_package.yaml:10]: done

**Review by DementevNikita — APPROVED** [review]

**Pull request merged**